content/en/docs/tasks/traffic-management/egress/egress-kubernetes-services/index.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Kubernetes Services for Egress Traffic
+description: Shows how to configure Istio  Kubernetes External Services.
```

Comment:
configure Istio with Kubernetes External Services?

Reply:
Addressed in https://github.com/istio/istio.io/pull/6206/commits/98616edc99a42e8fb148d152819c72d7bd222915.